pages/store.php

```diff
@@ -311,12 +312,13 @@ function store() {
 				$count++;
 
 				echo "<tr class='table_multicolumns'>
-					<td style='width:30%;'><a href='$self_link&view=jutsu&view_jutsu=$id'>{$jutsu['name']}</a></td>
-					<td style='width:25%;text-align:center;'>" . ucwords(str_replace('_', ' ', $jutsu['effect'])) . "</td>
-					<td style='width:25%;text-align:center;'>" . ucwords(str_replace('_', ' ', $jutsu['jutsu_type'])) . "</td>
-					<td style='width:26%;text-align:center;'>&yen;{$jutsu['purchase_cost']}</td>
-					<td style='width:28%;text-align:center;'>
-						<a href='$self_link&view=jutsu&purchase_jutsu={$jutsu['jutsu_id']}&jutsu_type={$jutsu['jutsu_type']}'>Purchase</a></td>
+					<td style='width:30%; text-align:center;'><a href='$self_link&view=jutsu&view_jutsu=$id'>{$jutsu['name']}</a></td>
+					<td style='width:25%; text-align:center;'>" . ucwords(str_replace('_', ' ', $jutsu['effect'])) . "</td>
+					<td style='width:25%; text-align:center;'>" . ucwords(str_replace('_', ' ', $jutsu['jutsu_type'])) . "</td>
+					<td style='width:25%; text-align:center;'>" . ucwords(str_replace('_', ' ', $jutsu['element'])) . "</td>
```

Comment:
```suggestion
					<td style='width:25%; text-align:center;'>" . System::unSlug($jutsu['effect']) . "</td>
					<td style='width:25%; text-align:center;'>" . System::unSlug($jutsu['jutsu_type']) . "</td>
					<td style='width:25%; text-align:center;'>" . $jutsu['element'] . "</td>
```
We have a helper function for this thankfully - Though for element I don't think it needs it (element is always one word, the idea of the unSlug is to go from a slug format (lower case with underscore instead of space) to normal, e.g. `residual_damage` => "Residual Damage"